### PR TITLE
New php versions

### PR DESCRIPTION
**Pull request by piotrooo**
Travis configuration for PHP 7.1, 7.2 and 7.3.
Bumped phpunit.
Fixed tests.

Issue #74.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
+  "license": "Apache-2.0",
+  "minimum-stability": "stable",
+  "authors": [
+    {
+      "name": "José Carlos Chávez",
+      "email": "jcchavezs@gmail.com"
+    }
+  ],
+  "require": {
+    "php": "^7.1"
```
Comment (sergeyklay) [inline composer.json]: Changing the minimum version is the breaking changes. I'm pretty sure users will be surprised. Why did you decide to drop PHP 7.0 support. Could you refer the issue where it was discussed?

Comment (piotrooo) [reply composer.json]: Here https://gitter.im/opentracing/opentracing-php?at=5c9bd54e36704f6edde2bb99 and following messages. 
Consider also: https://www.php.net/supported-versions.php

**Comment by jcchavezs** [comment]
I agree, we can get rid of 7.0. Since the only more or less used
instrumentation using opentracing is
https://github.com/jcchavezs/zipkin-php-opentracing and zipkin-php is soon
going to use php 7.1 (see [this](
https://github.com/openzipkin/zipkin-php#zipkin-instrumentation-for-php)).

Also DDTrace is not OpenTracing native:
https://github.com/DataDog/dd-trace-php/commit/237e3b60773d3ba7cbafaa8d6652dd0f0e7c7985#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780
but provides a compatible bridge so users can decide to use OT or not.

cc @pawelchcki @ellisv

tor. 4. apr. 2019 kl. 13:06 skrev Piotr Olaszewski <notifications@github.com

File: composer.json (hunk 1 above)
Comment (sergeyklay) [reply composer.json]: @piotrooo I'm aware of supported php versions. I just pointed that there is no reference to issue (not chat).

Comment (piotrooo) [reply composer.json]: @sergeyklay because there isn't any. What do you suggest?

Comment (sergeyklay) [reply composer.json]: Create an issue and describe the problem and the necessary changes. Then add link to the issue in PR description,

Comment (piotrooo) [reply composer.json]: There was some issue and I linked it.

Comment (sergeyklay) [reply composer.json]: Thank you

**Comment by piotrooo** [comment]
@jcchavezs any news?

**Comment by jcchavezs** [comment]
I al going to merge this if no further complaints.

lør. 6. apr. 2019, 08:49 skrev Piotr Olaszewski <notifications@github.com>:

**Pull request merged**